src/test/resources/application.conf

```diff
@@ -10,6 +10,10 @@ mantis {
 
   network.peer.max-peers = 3
```

Comment:
This one then becomes `max-outgoing-peers` right? Maybe it's worth renaming it just to make it more clear

Reply:
ps: Same comment for `src/test/resources/application.conf`